Chap_API_Struct.tex

```diff
@@ -169,7 +169,7 @@ \subsection{PMIx Return Status Constants}
 suitable connection either has not been or can not be made.
 %
 \declareconstitem{PMIX_ERR_BAD_PARAM}
-An incorrect parameter was passed to a \ac{PMIx} \ac{API}.
+An incorrect or unsupported parameter was passed to a \ac{PMIx} \ac{API}.
```

Comment:
This gets confusing when compared with the `PMIX_ERR_NOT_SUPPORTED` error code. If an unsupported parameter was passed then it should

Reply:
Since we have `PMIX_ERR_PARAM_VALUE_NOT_SUPPORTED` now, I don't think we want this change to the `PMIX_ERR_BAD_PARAM` definition.